pkg/kv/kvserver/replica.go

```diff
@@ -248,47 +248,52 @@ type Replica struct {
 		stateMachine replicaStateMachine
 		// decoder is used to decode committed raft entries.
 		decoder replicaDecoder
+	}
```

Comment:
Mind sending this commit separately? It's an easy merge.

Reply:
Done in https://github.com/cockroachdb/cockroach/pull/95861.